_posts/2019-01-09-tools-we-use.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "Tools We Use at Ombu Labs"
+date: 2019-01-09 10:00:00
+categories: ["agile"]
+author: "emily"
+---
+
+At Ombu Labs, we have a number of tools that help us to keep organized, write better code and communicate more effectively with our clients.
+
+<!--more-->
+
+### Daily Work
+- We use [Slack](https://ombulabs.slack.com/) to chat with team members while working.
+- We use [Trello](http://trello.com) to manage our clients and leads.
```

Comment:
@ercohen14 It might sound a little confusing to the reader. We do use Trello to manage the status of our sales pipeline and client projects.